superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/sharedControls.ts

```diff
@@ -55,36 +55,4 @@ export const headerFontSize: CustomControlItem = {
   },
 };
 
-export const subheaderFontSize: CustomControlItem = {
```

Comment:
Ahh... removing it from SharedControls makes sense, but doesn't this remove it from the regular BigNumber, too? That one _does_ have the Subheader, and _should_ have the control.